build/config/compiler/BUILD.gn

```diff
@@ -642,6 +642,15 @@ if (is_win) {
     "/wd4311",  # Pointer truncation from PVOID to DWORD.
     "/wd4477",  # Format string requires wchar_t*
   ]
+  import("//build/toolchain/goma.gni")
+  if (is_clang && use_goma) {
+    # goma compilation doesn't preserve file case, so don't warn about
+    # case-mismatches.
+    default_warning_flags += [
+      "-Wno-nonportable-include-path",
```

Comment:
On second thought, why not just replace add this unconditionally on Windows? It feels odd that the TU will get different flags based on whether or not goma is used. You call though.

Reply:
The issue is specific to goma AFAICT, and leaving it this way means we'll still get warnings locally for legitimate case mismatches. It seems like it would be a shame to have no way of getting warnings for things that are legitimately incorrect.